src/commands/sdm.ts

```diff
@@ -128,13 +128,21 @@ export class SdmCommand implements MessageCommand, ApplicationCommand {
     }
 
     async handleMessage(message: ProcessableMessage, client: Client<boolean>): Promise<CommandResult> {
-        const args = substringAfter(message.cleanContent, this.name).trim().split(/\s+/g).filter(s => !!s);
+        const replyRef = message.reference?.messageId;
+        const isReply = replyRef !== undefined;
+        let args = substringAfter(message.cleanContent, this.name).trim().split(/\s+/g).filter(s => !!s);
 
-        if (!args.length) {
+        if (!args.length && !isReply) {
             await message.reply("Bruder da ist keine Frage :c");
             return;
         }
 
+        if (isReply) {
+            // throw in everything in a single line
+            // shadow style :D
+            args = substringAfter((await message.channel.messages.fetch(replyRef!)).content, this.name).trim().split(/\s+/g).filter(s => !!s);
+        }
+
         let question = args.join(" ").replace(/\s\s+/g, " ");
```

Comment:
Nutz doch hier das question anstatt programmatisch args wieder zusammenzusetzen. Das ist auch eh schon mutable

Reply:
sehe keinen Grund wo das besser sein sollte..

Reply:
1. Jetzt sind zwei Variablen let und nicht const - ist aber nichtmal so wild
2. Du baust dir ein args Array aus einer Nachricht zusammen, nur um es danach wieder in den string zu joinen, der ohnehin in der Nachricht stand. Unnötig. 
3. Semantik von args verändert

Reply:
ok